tooling/azure-automation/resources-cleanup/src/clean-orphaned-role-assignments.ps1

```diff
@@ -1,19 +1,23 @@
 Param
 (
   [Parameter (Mandatory= $false)]
-  [System.Boolean] $dryRun = $false
+  [System.Boolean] $dryRun = $false,
+  [Parameter (Mandatory= $false)]
+  [string] $SubscriptionId = "1d3378d3-5a3f-4712-85a1-2485495dfc4b",
+  [Parameter (Mandatory= $false)]
+  [string] $ManagedIdentityId
 )
```

Comment:
Is there a calling script/module that needs to be updated with these parameters?